src/typeguard/_checkers.py

```diff
@@ -29,6 +29,8 @@
     Union,
 )
 
+import typing_extensions
```

Comment:
You need to follow existing conventions. Add this conditional import to section after unconditional imports:
```suggestion
if sys.version_info >= (3, 11):
    from typing import NotRequired, Required
else:
    from typing import NotRequired, Required
```

Reply:
You marked this as unresolved even though you didn't make any changes, why?